mappings/net/minecraft/server/world/ServerWorld.mapping

```diff
@@ -62,18 +66,30 @@ CLASS net/minecraft/class_3218 net/minecraft/server/world/ServerWorld
 	METHOD method_14175 addEntity (Lnet/minecraft/class_1297;)Z
 		ARG 1 entity
 	METHOD method_14176 save (Lnet/minecraft/class_3536;ZZ)V
+		COMMENT Saves the world.
 		ARG 1 progressListener
+			COMMENT the listener for the saving process, or {@code null} to specify none
 		ARG 2 flush
+			COMMENT if it should immediately write all data to storage device
 		ARG 3 savingDisabled
+			COMMENT if true, this method does nothing.
 	METHOD method_14177 isInBlockTick ()Z
 	METHOD method_14183 getStructureManager ()Lnet/minecraft/class_3485;
 	METHOD method_14188 saveLevel ()V
 	METHOD method_14189 tickBlock (Lnet/minecraft/class_2338;Lnet/minecraft/class_2248;)V
 		ARG 1 pos
 		ARG 2 block
 	METHOD method_14190 getEntity (Ljava/util/UUID;)Lnet/minecraft/class_1297;
+		COMMENT {@return the entity using the UUID, or {@code null} if none was found}
+		COMMENT
+		COMMENT @see World#getEntityById
 		ARG 1 uuid
 	METHOD method_14191 sendToPlayerIfNearby (Lnet/minecraft/class_3222;ZDDDLnet/minecraft/class_2596;)Z
+		COMMENT Sends the {@code packet} to {@code player} if the player
+		COMMENT is near the provided coordinates. Returns true if successful.
```

Comment:
This should be documented using a `@return` tag and explain what constitutes a success.

Reply:
```suggestion
		COMMENT is near the provided coordinates.
		COMMENT
		COMMENT @return whether the packet was sent
```